### DISCUSSION TITLE
Adding support for more types of updates issues.

### DISCUSSION THREAD

**Discussion by theofficialvedantjoshi** [david-lev/pywa#72] [General]
I have started building updates for things such as account alerts, phone_number_quality_updates, template_category_updates, etc. but I am facing an issue in the WhatsApp cloud api docs.
Many of the Enums have no specified value in the WhatsApp docs and without that its hard to create filters or specific messages and data models to show the user. Any way I can find these Enums anywhere so I can contribute.
@david-lev 

Thank you

**Comment by yehuda-lev** [comment]
Hi, thanks!
We also tried to add it but the docs are not clear enough 

To send an unreal 'update' - it is not good because in reality a different update is received

 If you can build it according to a real update received on your computer that would be great!

You can test it by `on_raw_update`

Also note to add the update to the [test file](https://github.com/david-lev/pywa/tree/master/tests%2Fdata%2Fupdates%2F18.0)

 Thank you for your help 🙏
